src/backend/main.ts

```diff
@@ -1030,6 +1030,27 @@ ipcMain.handle(
         rmSync(winePrefix, { recursive: true })
       }
     }
+    if (shouldRemoveSetting) {
+      logInfo(`Removing ${appName.concat('.json')}`, {
+        prefix: LogPrefix.Backend
+      })
+      // remove setting json if exists
+      if (existsSync(join(heroicGamesConfigPath, appName.concat('.json')))) {
```

Comment:
maybe this should be a variable:
`const gameSettingsFile = join(heroicGamesConfigPath, appName.concat('.json'))`